src/main/java/appeng/integration/IntegrationType.java

```diff
@@ -50,7 +51,14 @@ public IIntegrationModule createInstance()
 		}
 	},
 
-	InvTweaks( IntegrationSide.CLIENT, "Inventory Tweaks", "inventorytweaks" ),
+	InvTweaks( IntegrationSide.CLIENT, "Inventory Tweaks", "inventorytweaks" )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00115)